splice_machine/src/main/java/com/splicemachine/pipeline/IndexWriteHandler.java

```diff
@@ -33,6 +33,7 @@
  *         Created on: 5/1/13
  */
 public class IndexWriteHandler extends RoutingWriteHandler{
+    // todo: IndexWriteHandler is the only usage of RoutingWriteHandler. Merge into one class
```

Comment:
not sure we want to do this, there is probably a design decision behind that, we have other  few examples where an interface / abstract class is implemented by a single concrete class.

Reply:
i have to remove this comment. Reason is that the IndexWriteHandler is in another component where the RoutingWriteHandler hasn't access to.